doc/doc-site/docs/resources/audit.md

```diff
@@ -4,4 +4,10 @@ title: 🧑‍💻 Audit
 hide_title: false
 ---
 
-Coming soon!
+The emojicoin dot fun  Move package is audited:
```

Comment:
There are two spaces between `fun` and `Move`.

Reply:
Addressed in 9a5aa1